examples/surface_timeseries_.py

```diff
@@ -6,6 +6,12 @@
 
 .. tags:: experimental
 """
+try:
+    from packaging.version import parse
+except ModuleNotFoundError:
+    raise ModuleNotFoundError(
+        "You must have packaging installed to run this example."
```

Comment:
I like it when error messages have the format: something went wrong, here's how you might fix it

Can this error include a suggestion to use "pip install packaging" (and/or conda)